ui/components/app/whats-new-popup/whats-new-popup.js

```diff
@@ -9,21 +9,13 @@ import {
   MetaMetricsEventName,
 } from '../../../../shared/constants/metametrics';
 import {
-  ///: BEGIN:ONLY_INCLUDE_IF(blockaid)
-  NOTIFICATION_BLOCKAID_DEFAULT,
```

Comment:
Should all instances of this const be removed from the cod base? It's still present in tests

![Screenshot 2024-06-11 at 8 39 05 AM](https://github.com/MetaMask/metamask-extension/assets/8112138/345982d3-0bf7-434a-aab4-e9ae8c140b14)